### ISSUE TITLE
official docker image: possible memory leak?

### ISSUE THREAD

**Issue by acidzebra** [stefanwichmann/kelvin#48]
Hey Stefan,

first I would like to thank you for this amazing software 👍 

Second, I grabbed kelvin:latest from the docker registry and run it on my Synology NAS. This works well, at startup the container consumes about 6MB RAM but after ~8 hours this has increased to nearly 200MB and looks to be increasing still. I'm wondering if there is some kind of memory leak or whether this is known behavior and will top out eventually. My NAS is limited on RAM so I'm a bit worried.

cheers
AZ

**Comment by stefanwichmann** [comment]
Hi Michiel,

thanks for reporting this. Issue #45 also states high resource usage. So I should have a look at this... I'll keep you posted!

**Comment by acidzebra** [comment]
I've observed it for longer and may have piped up too soon - the container will bounce between 100 and ~300 MB happily and not grow larger. I still think that's quite a lot of memory use (my NAS is unfortunately limited) but I've set a hard RAM limit at 300MB and the container to auto-restart and that seems to have done it. And the software is very much worth it, very enjoyable! 

I've not seen the high CPU usage reported in that other issue but Synology is Intel Atom and Pi is ARM, IIRC, so apples and oranges IMO.

I will close out this issue.
